pkgs/development/python-modules/typed-settings/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptoolsBuildHook
+, attrs
+, toml
+, pytestCheckHook
+, click
+}:
+
+buildPythonPackage rec {
+  pname = "typed-settings";
+  version = "0.9.2";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "203c1c6ec73dd1eb0fecd4981b31f8e05042f0dda16443190ac9ade1113ff53d";
+  };
+
+  nativeBuildInputs = [
+    setuptoolsBuildHook
+    pytestCheckHook
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    toml
+  ];
+
+  checkInputs = [
+    click
+  ];
+
+  meta = {
+    description = "Typed settings based on attrs classes";
+    homepage = "https://gitlab.com/sscherfke/typed-settings";
+    license = lib.licenses.mit;
+    maintainer = with lib.maintainers; [ fridh ];
```

```suggestion
    maintainers = with lib.maintainers; [ fridh ];
```
Could you change to this?

It might be too late to write here, if so I will open a new issue/PR.

---

oops, thank you for mentioning it. Pushed d4d1a000be22419dbe1f76f3f30bc3f52cb10d44